content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -53,47 +61,65 @@ The following commands have relative references in the Istio directory. You must
 
 Choose this option if your cluster doesn't have [Tiller](https://github.com/kubernetes/helm/blob/master/docs/architecture.md#components) deployed and you don't want to install it.
 
-1. Install all the Istio's [Custom Resource Definitions or CRDs for short](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) via `kubectl apply`, and wait a few seconds for the CRDs to be committed to
-the Kubernetes API server:
+1. Make an Istio working directory for fetching the charts:
+
+    {{< text bash >}}
+
+    $ mkdir -p $HOME/istio-fetch
+
+    {{< /text >}}
+
+1. Fetch the helm templates needed for installation:
 
     {{< text bash >}}
-    $ for i in install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl apply -f $i; done
+
+    $ helm fetch istio.io/istio-init --untar --untardir $HOME/istio-fetch
+    $ helm fetch istio.io/istio --untar --untardir $HOME/istio-fetch
+
     {{< /text >}}
 
-1. Render Istio's core components to a Kubernetes manifest called `istio.yaml`:
+1. Create a namespace for the `istio-system` components:
 
     {{< text bash >}}
-    $ cat @install/kubernetes/namespace.yaml@ > $HOME/istio.yaml
-    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system >> $HOME/istio.yaml
+
+    $ kubectl create namespace istio-system
+
     {{< /text >}}
 
-    If you want to enable [global mutual TLS](/docs/concepts/security/#mutual-tls-authentication), set `global.mtls.enabled` and `global.controlPlaneSecurityEnabled` to `true` for the last command:
```

Comment:
note I removed this - this doesn't belong in the quickstart, but instead in some more "advanced helm guide" or in the as-of-yet unimplemented profiles guide.

@linsun ^^